tasks/podman-setup.yaml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Configure loginctl to prevent logout causing podman to exit
+  ansible.builtin.shell: loginctl enable-linger
+
+- name: Set podman variable
+  set_fact:
+    podman_conf: podman.conf
+
+- name: "Install Podman on RHEL8 server"
+  package:
+    name:
+      - "@container-tools" # install module 'container-tools'
+      - podman-docker
```

Comment:
We're going to use `podman`, `podman-docker` provides the ability to run `docker` commands but talk to the podman socket, which is not used by us.